src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -95,11 +101,15 @@ static void DeleteOtherPlatformHost(Permutation permutation)
                 x86.Delete();
             }
         }
-
-        [TestFixtureSetUp]
-        public void RunBeforeAnyTests()
+    }
+    
+    [SetUpFixture]
+    public class TestSessionInitializer
+    {
+        [SetUp]
+        public void Setup()
         {
-            SessionId = Guid.NewGuid().ToString();
+            Base.SessionId = DateTime.UtcNow.Ticks.ToString();
```

Comment:
Also you can filter them by category and description + latest run. e.g. show me the test result of the latest build where test category is performance or test description is 'Audit forwarding On / Off'.